frontend/lib/ui/artefact_page/issue_attachments/issue_attachment_widget.dart

```diff
@@ -47,7 +48,11 @@ class IssueAttachmentWidget extends StatelessWidget {
                 ? 'Attached by an enabled attachment rule'
                 : 'Attached by a disabled attachment rule',
             child: InkWell(
-              onTap: () {}, // Placeholder link
+              onTap: () => navigateToIssuePage(
```

Comment:
This makes this button work:

<img width="1901" height="941" alt="Image" src="https://github.com/user-attachments/assets/96c7cb2f-2be5-4a5d-9619-92400ae62d2a" />